docs/pages/router/advanced/drawer.mdx

```diff
@@ -32,15 +33,45 @@ module.exports = {
 ```
 
 > After changing your `babel.config.js`, once run `npx expo start --clear` to start the development server and clear the babel cache.
```

Comment:
```suggestion
> After changing your **babel.config.js**, run `npx expo start --clear` once to start the development server and clear the babel cache.
```

Not related to PR (happy to create a follow up one too if required) but this is a small suggestion to apply our writing style guide for filenames and also update the verbiage about running the command as an instruction.